ofrak_core/ofrak/gui/server.py

```diff
@@ -166,6 +168,27 @@ def __init__(
         self._job_ids: Dict[str, bytes] = defaultdict(
             lambda: ofrak_context.id_service.generate_id()
         )
+        try:
+            import aiohttp_cors
+
+            # From: https://github.com/aio-libs/aiohttp-cors
+            # Configure default CORS settings.
+            cors = aiohttp_cors.setup(
+                self._app,
+                defaults={
+                    "*": aiohttp_cors.ResourceOptions(
+                        allow_credentials=True,
+                        expose_headers="*",
+                        allow_headers="*",
+                    )
+                },
+            )
+
+            # Configure CORS on all routes.
+            for route in list(self._app.router.routes()):
+                cors.add(route)
+        except ImportError:
+            pass
```

Comment:
This `try/except` block should probably be inside a conditional/flag. Wrapping all the routes with CORS is great for local debugging, but runs the risk of causing problems when the server is subclassed and routes are added. 

Also, if we're going to rely on `aiohttp_cors`, we need to make sure to add the dependency to the list.

Reply:
Added the dependency to `requirements.txt`.

What would the use of `aiohttp_cors` be conditional upon? Check whether the type of self is `AiohhtpOFRAKServer`?

Reply:
That could be one option. Or there could be an additional argument `use_cors` passed in `__init__` with a default value of `False` that a user could turn on when debugging. I'm open to other options, but I don't think CORS should be on by default.

Reply:
Is CORS just for debugging though? At this point, I'm not able to run the GUI without CORS on any of Firefox, Safari, or Chrome. Perhaps that is indicative of something else needing to be fixed?

Reply:
Yes, CORS should just be for debugging. You should be able to run the GUI without CORS as long as the GUI is built and copied into the folder that the server serves static files from. CORS is only necessary when running a separate GUI server on a different port. This should never happen in "production" and should only be for debugging.

Reply:
Should this option be made visible as a new CLI argument for the gui?

Reply:
Might as well be a new argument, I think that's a good idea.